.github/workflows/doc.yaml

```
@@ -0,0 +1,37 @@
+# Copyright 2024 Adevinta
+
+name: Documentation
+on:
+  push:
+    branches:
+    - mdBook
```

Should we use `main` here? Check file comment for more details.

```suggestion
      - main
```

---

This is included in the last commit which means to be dropped before the merging. It's only to test the changes in this branch.